lte/gateway/python/integ_tests/federated_tests/docker/run.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse
+import os
+import subprocess
+
+SNOWFLAKE_PATH = '../../../../../../.cache/feg/'
+SNOWFLAKE_FILE = '../../../../../../.cache/feg/snowflake'
+
+
+def main() -> None:
+    args = _parse_args()
+    if not os.path.isfile(SNOWFLAKE_FILE):
+        _create_snowflake_file()
+    _exec_docker_cmd(args)
+
+
+def _create_snowflake_file() -> None:
+    if os.path.isdir(SNOWFLAKE_FILE):
+        _exec_cmd(['rm', '-r', SNOWFLAKE_FILE])
+    print("Creating snowflake file")
+    _exec_cmd(['mkdir', '-p', SNOWFLAKE_PATH])
+    _exec_cmd(['touch', SNOWFLAKE_FILE])
```

Comment:
At what point does the file get filled with a UUID?

Reply:
To my understanding that is done by the magmad services on the feg via the snowflake.py package, i.e. not anywhere in the touched code. 
You can check that by e.g. replacing the snowflake file with a directory and checking the logs of the magmad container on the feg. It will throw an error:
```
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.8/dist-packages/magma/magmad/main.py", line 316, in <module>
    main()
  File "/usr/local/lib/python3.8/dist-packages/magma/magmad/main.py", line 61, in main
    logging.info('Starting magmad for UUID: %s', snowflake.make_snowflake())
  File "/usr/local/lib/python3.8/dist-packages/snowflake.py", line 43, in make_snowflake
    _write_new_id(snowflake_file)
  File "/usr/local/lib/python3.8/dist-packages/snowflake.py", line 33, in _write_new_id
    f = open(snowflake_file, "w")
IsADirectoryError: [Errno 21] Is a directory: '/etc/snowflake'
```

Reply:
Yes, it's filled by the Python library if there's no folder there.

Reply:
Great, thanks!